plantdescriptionengine/pom.xml

```diff
@@ -0,0 +1,168 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>eu.arrowhead</groupId>
+        <artifactId>core</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>arrowhead-plantdescriptionengine</artifactId>
+    <packaging>jar</packaging>
+    <name>Arrowhead Plant Description Engine</name>
+    <description>Arrowhead Plant Description Engine core service</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>11</maven.compiler.release>
+        <kalix.version>0.6.0</kalix.version>
+        <mysql.jdbc.driver>8.0.16</mysql.jdbc.driver>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>se.arkalix</groupId>
+            <artifactId>kalix-base</artifactId>
+            <version>${kalix.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>se.arkalix</groupId>
+            <artifactId>kalix-core-plugins</artifactId>
+            <version>${kalix.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>se.arkalix</groupId>
+            <artifactId>kalix-dto-processor</artifactId>
+            <version>${kalix.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>se.arkalix</groupId>
+            <artifactId>kalix-dto</artifactId>
+            <version>${kalix.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.1.52.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>1.4.200</version>
+        </dependency>
```

Comment:
if you using this only in tests, then the dependency is already in the parent pom.

Reply:
Removed it.